.buildkite/scripts/steps/serverless.ps1

```diff
@@ -0,0 +1,94 @@
+function serverless_up {
+  Write-Output "~~~ Starting Serverless Observability project"
+
+  $Workspace = & git rev-parse --show-toplevel
+  $TfDir = Join-Path -Path $Workspace -ChildPath "test_infra/serverless/"
+
+  $Env:EC_API_KEY = Retry-Command -ScriptBlock {
+    vault kv get -field=apiKey kv/ci-shared/platform-ingest/platform-ingest-ec-prod
+  }
+
+  if (-not $Env:EC_API_KEY) {
+      Write-Error "Error: Failed to get EC API key from vault"
+      exit 1
+  }
+
+  Push-Location -Path $TfDir
+  & terraform init
+  & terraform apply -auto-approve
+
+  $Env:ELASTICSEARCH_HOST = & terraform output -raw es_host
+  $Env:ELASTICSEARCH_USERNAME = & terraform output -raw es_username
+  $Env:ELASTICSEARCH_PASSWORD = & terraform output -raw es_password
+  $Env:KIBANA_HOST = & terraform output -raw kibana_endpoint
+  $Env:KIBANA_USERNAME = $Env:ELASTICSEARCH_USERNAME
+  $Env:KIBANA_PASSWORD = $Env:ELASTICSEARCH_PASSWORD
```

Comment:
What's the reason for no using `KIBANA_USERNAME` instead but `ELASTICSEARCH_USERNAME`?